backend/expense-calculator/src/main/java/org/egov/digit/expense/calculator/util/ExpenseCalculatorUtil.java

```diff
@@ -139,11 +139,11 @@ public List<Bill> fetchBills(RequestInfo requestInfo, String tenantId, String co
         List<String> billIds = expenseCalculatorRepository.getBills(contractId, tenantId);
 
         StringBuilder url = searchURI(configs.getBillHost(), configs.getExpenseBillSearchEndPoint());
-        //Pagination pagination = Pagination.builder().limit(100).offSet(0).order(Order.ASC).build();
+        Pagination pagination = Pagination.builder().limit(100).offSet(0).order(Order.ASC).build();
```

Comment:
keep a pagination object in search criteria, and change the offset and limit to a environment variable

Reply:
Changes implemented in new commits.